src/__tests__/CompareResults/beta/CompareResultsView.test.tsx

```diff
@@ -1,10 +1,10 @@
-import CompareResultsViewBeta from '../../../components/CompareResults/beta/CompareResultsView';
+import ResultsViewBeta from '../../../components/CompareResults/beta/ResultsView';
```

Comment:
For consistency we should use `ResultsViewBeta` or `CompareResultsViewBeta` when importing. I'm ok with either version.

```suggestion
import CompareResultsViewBeta from '../../../components/CompareResults/beta/ResultsView';
```